### ISSUE TITLE
Implement a simple engine server that receives StartEngine commands

### ISSUE THREAD

**Issue by thrau** [cognitivexr/CogStream#38]
To enable decentralized control of engines, we need the mediator to send StartEngine commands to other nodes, rather than starting the engines on the host which the mediator runs on. To that end, we should create a small HTTP server application that wraps the StartEngine command of the runtime https://github.com/cognitivexr/CogStream/blob/1edfc6b385660c20afbd48eb13f67fe3055cbd73/mediator/pkg/runtime/runtime_plugin.go#L155

basically the mediator has a controller/worker architecture, where a client requesting an engine talks to the controller, and the controller talks to the workers that spawn engines.

**Comment by thrau** [comment]
related to #33

**Comment by thrau** [comment]
actually, just having the controller being a websocket proxy that manages several mediator instances may be easier. since the mediator has a bunch of logic to find/control engines that exist locally on the file system.

**Comment by thrau** [comment]
taking this :-)
